lib/css/components/post-summary.less

```
@@ -158,7 +160,14 @@
             font-weight: 500;
         }
 
+        &.s-badge {
+            font-size: var(--fs-body1);
+            line-height: var(--lh-md);
+            padding: var(--su2) var(--su4);
+        }
+
         &.has-answers,
+        // TODO: Remove .has-bounty styling
```

nit: Adding a specific label here will allow us to quickly and trivially find all deprecated functionality when we eventually do a new major/breaking release.

```suggestion
        // TODO DEPRECATED: Remove .has-bounty styling
```